src/vellum/workflows/nodes/core/map_node/node.py

```diff
@@ -88,6 +98,13 @@ def run(self) -> Outputs:
 
         return self.Outputs(**mapped_items)
 
+    def _context_run_subworkflow(
+        self, *, item: MapNodeItemType, index: int, parent_context: Optional[ParentContext] = None
+    ) -> None:
+        parent_context = parent_context or self._context.parent_context
+        with execution_context(parent_context=parent_context):
+            self._run_subworkflow(item=item, index=index)
+
     def _run_subworkflow(self, *, item: MapNodeItemType, index: int) -> None:
         subworkflow = self.subworkflow(parent_state=self.state, context=self._context)
```

Comment:
[blocking] we'll need to do a change similar to [this file](https://github.com/vellum-ai/vellum-python-sdks/pull/294/files#diff-b1ecb01085162e6bb71295fa2b7372eb706f77bb97f45cae52809f3faec67f6e) in order for the inner subworkflow events to pipe through the api. I suggest we do so that we can make assertions about those events' parent context in the test